docs/modules/debug/network-analyzer.md

```diff
@@ -0,0 +1,36 @@
+<%( TOC )%>
+
+# Network Analyzer window
+
+The Network Analyzer window is an Unity Editor window which displays live bandwidth usage for a worker broken down by component type. This window can assist you in finding networking bottlenecks in your game.
+
+## Usage
+
+To open the window, ensure the `Debug` package is added to your project and select **SpatialOS** > **Window** > **Network Analyzer** from the Unity Editor menu.
+
+Enter Play mode to start the data collection.
+
+## Explanation
+
+The window lists all SpatialOS components in your project and displays bandwidth usage for component update ops for each component. The data shown is collected over 60 frames and then normalized to show an approximate reading per second.
+
+<img src="{{assetRoot}}assets/modules/debug/network-analyzer.png" style="margin: 0 auto; width: auto; display: block;" />
+
+### Metrics
+
+* `Op/s in` - This metric describes the number of component update ops that this worker is receiving per second.
+* `KB/s in` - This metric describes the bandwidth used by component updates that this worker is receiving per second.
+* `Op/s out` - This metric describes the number of component update ops that this worker is sending per second.
+* `KB/s out` - This metric describes the bandwidth used by component updates that this worker is sending per second.
```

Comment:
might look nicer as a table

Reply:
Hm maybe - it would only have two columns though. Let me mock it up

Reply:
Yeah looks better